cmd/fleet/cron.go

```diff
@@ -190,32 +191,52 @@ func cronVulnerabilities(
 				appConfig = freshAppConfig
 			}
 
-			var vulnAutomationEnabled bool
+			var vulnAutomationEnabled string
 
-			// only one of the webhook or a jira integration can be enabled at a
+			// only one vuln automation (i.e. webhook or integration) can be enabled at a
 			// time, enforced when updating the appconfig.
 			if appConfig.WebhookSettings.VulnerabilitiesWebhook.Enable {
-				vulnAutomationEnabled = true
-			} else {
-				for _, intg := range appConfig.Integrations.Jira {
-					if intg.EnableSoftwareVulnerabilities {
-						vulnAutomationEnabled = true
-						break
+				vulnAutomationEnabled = "webhook"
+			}
+			// check for jira integrations
+			for _, j := range appConfig.Integrations.Jira {
+				if j.EnableSoftwareVulnerabilities {
+					if vulnAutomationEnabled != "" {
+						err := errors.New("more than one automation enabled: jira check")
+						level.Error(logger).Log("err", err)
+						sentry.CaptureException(err)
 					}
+					vulnAutomationEnabled = "jira"
+					break
 				}
 			}
+			// check for zendesk integrations
+			for _, z := range appConfig.Integrations.Zendesk {
+				if z.EnableSoftwareVulnerabilities {
+					if vulnAutomationEnabled != "" {
+						err := errors.New("more than one automation enabled: zendesk check")
```

Comment:
@noahtalerman, Should "zendesk" be capitalized here? I don't want to send unnecessary PRs for such things if I'm incorrect.